PacketDefinitions420/PacketHandlerManager.cs

```diff
@@ -62,7 +62,7 @@ internal void InitializePacketConvertors()
                 {
                     if (attr is PacketType)
                     {
-                        if (((PacketType)attr).ChannelId == Channel.CHL_LOADING_SCREEN)
+                        if (((PacketType)attr).ChannelId == Channel.CHL_LOADING_SCREEN || ((PacketType)attr).ChannelId == Channel.CHL_COMMUNICATION)
```

Comment:
When this `Channel.CHL_COMMUNICATION` channel used? If it is used in both LoadScreen and Game as I suspect (I think it is used for ingame chat) then this solution will disable chat?

Reply:
Chat is currently disabled because convertor does not support the communication channel. This PR fixes that.

Reply:
Just to make sure:
There are no Game packets in this communication channel and all the packets there are LoadingScreen ones?

Reply:
Yes, the communication channel is only for LoadScreen packets. Bit confusing I know, but it contains the chat packets.